cmd/adapter-rest/startcmd/start.go

```diff
@@ -98,6 +100,11 @@ const (
 		"Alternatively, this can be set with the following environment variable: " + staticFilesPathEnvKey
 	staticFilesPathEnvKey = "ADAPTER_REST_STATIC_FILES"
 
+	outputDescriptorsFilePathFlagName  = "output-descriptors-path"
```

Comment:
mention is as 'output descriptors of credential manifests supported by the adapter'

Reply:
done